### ISSUE TITLE
Support organization owner in Github api url

### ISSUE THREAD

**Issue by PandaXass** [MonolithProjects/ansible-github_actions_runner#23]
When creating a registration token for self-hosted runner through Github API, the `{owner}` can be an organization or simply another Github account instead of the Github account which owns the access token.
https://docs.github.com/en/rest/reference/actions#create-a-registration-token-for-a-repository

For example:
```
curl \
  -X POST \
  -H "Accept: application/vnd.github.v3+json" \
  -u bob:access_token
  https://api.github.com/repos/myorg/hello-world/actions/runners/registration-token
```
Or
```
curl \
  -X POST \
  -H "Accept: application/vnd.github.v3+json" \
  -u bob:access_token
  https://api.github.com/repos/alice/hello-world/actions/runners/registration-token
```

And it is the same for "List self-hosted runners for a repository" endpoint.
https://docs.github.com/en/rest/reference/actions#list-self-hosted-runners-for-a-repository

Is it okay to introduce another variable to support above usecase? Thanks a lot in advance!
And I have created a PR for this issue.

**Comment by MonolithProjects** [comment]
Hi @PandaXass , adding the GitHub Actions runner to the organization repository by this Ansible role is supported even now. It is just necessary to use the user which is the Org owner or has admin rights to the repository. Simply use that user name as a value of `github_account` variable and the role should work fine.

**Comment by PandaXass** [comment]
Hi @MonolithProjects , thank you so much for your quick reply. I tried as what you said, but it doesn't seem to work.

I also updated my issue discription, especially for the 2nd example.

**Comment by MonolithProjects** [comment]
OK i see now what you mean

**Comment by MonolithProjects** [comment]
@PandaXass I corrected the authentication in the Ansible uri tasks to be only the token without the user name (anyway the username/password authentication which the role was using until now (with token instead of password) will be [deprecated soon](https://developer.github.com/changes/2020-02-14-deprecating-password-auth/). Basically now it is identical to:
```
curl \
-X POST \
-H "Authorization: token <access_token>" \
-H "Accept: application/vnd.github.v3+json" \
https://api.github.com/repos/alice/hello-world/actions/runners/registration-token
```
The change is in the [develop](https://github.com/MonolithProjects/ansible-github_actions_runner/tree/develop) branch. Could you test it if this resolved your issue?

**Comment by PandaXass** [comment]
Hi @MonolithProjects I can confirm it works!

**Comment by MonolithProjects** [comment]
Cool... happy to hear that. Issue is now fixed #26
